### PR TITLE
Removed invalid issuer field on valid credentials

### PR DESCRIPTION
**Pull request by PatStLouis**
There is still issues with negative issuer testing, and also testing if the issuer field can be an object.

This fixes the problem where all positive tests would send an invalid issuer.

Problematic tests include:
- Testing if an issuer can be an object
- Testing if an issuer can be a http uri
- Negative testing of empty issuer field
- All description and name issuer tests

### CODE REVIEW

**Review by aljones15 — APPROVED** [review]
did not look extensively do to the size, but this LGTM and feel free to approve. also +1 to multiple PRs for this.

File: tests/input/names-and-descriptions/credential-multi-language-description-ok.json [hunk 1]
```diff
@@ -15,9 +15,6 @@
     "@value":"Cane",
     "@language": "it"
   }],
-  "issuer": {
-    "id": "did:issuer:dog"
```
Comment (BigBlueHat) [inline tests/input/names-and-descriptions/credential-multi-language-description-ok.json:19]: 🐶 ❓ 😜

**Review by BigBlueHat — APPROVED** [review]
Thanks, @PatStLouis! Approving and then merging this change.

File: tests/input/names-and-descriptions/credential-multi-language-description-ok.json (hunk 1 above)
Comment (PatStLouis) [reply tests/input/names-and-descriptions/credential-multi-language-description-ok.json:19]: I saw this, easter egg maybe?

**Pull request merged**